core/app/models/spree/order.rb

```diff
@@ -27,6 +27,25 @@ def generate_number(options = {})
     alias :display_ship_total :display_shipment_total
     alias_attribute :ship_total, :shipment_total
 
+    MONEY_THRESHOLD  = 999_999
```

Comment:
We're using scale 10 precision 2 scale for our price fields so I believe `99_999_999.99` is large as we can go so `99_999_999`?